### PR TITLE
Linked templates documentation with Render Templates documentation.

### PR DESCRIPTION
**Pull request by dekiel**
Example usage with all local files added as it's most common used case.

**Description**

Changes proposed in this pull request:

- Add link to Render Template documentation in templates documentation.
- Add all local files example usage.

### CODE REVIEW

File: development/tools/cmd/rendertemplates/README.md [hunk 1]
```diff
@@ -160,6 +160,13 @@ go run development/tools/cmd/rendertemplates/main.go --data path/to/directory/wi
 
 By default, the Render Templates downloads `config.yaml` and templates files from [GitHub](https://github.com/kyma-project/test-infra). You can specify paths to the `config.yaml` file and templates directory on the command line to override defaults.
 
+To work with local files only provide all paths in flags.
```
Comment (NHingerl) [inline development/tools/cmd/rendertemplates/README.md:163]:
```suggestion
To work with local files only, provide all paths in flags.
```

File: development/tools/cmd/rendertemplates/README.md [hunk 2]
```diff
@@ -160,6 +160,13 @@ go run development/tools/cmd/rendertemplates/main.go --data path/to/directory/wi
...
+
+Example for test-infra repository.
```
Comment (NHingerl) [inline development/tools/cmd/rendertemplates/README.md:165]:
```suggestion
Example for test-infra repository:
```

File: templates/README.md [hunk 3]
```diff
@@ -2,7 +2,9 @@
 
 ## Overview
 
-Jobs and Prow configuration are generated from templates. The `templates` directory has the following structure:
+Jobs and Prow configuration are generated from templates by Render Templates tool. Check [**Render Templates documentation**](../development/tools/cmd/rendertemplates/README.md) for details about usage.
```
Comment (NHingerl) [inline templates/README.md:5]:
```suggestion
Jobs and Prow configuration are generated from templates by the Render Templates tool. Check the [Render Templates documentation](../development/tools/cmd/rendertemplates/README.md) for details about usage.
```

**Comment by dekiel** [comment]
@NHingerl corrected.

**Review by NHingerl — APPROVED** [review]

**Pull request merged**